ext/afform/core/Civi/Api4/Action/Afform/SubmitFile.php

```diff
@@ -98,6 +98,20 @@ protected function processForm() {
 
     $file = civicrm_api3('Attachment', 'create', $attachmentParams);
 
+    /**
+     * Updates contact record to use existing file table data for contact_image. Temporary fix for dev/core#4251
+     */
+    if ($attachmentParams['entity_table'] == 'civicrm_contact' && isset($attachmentParams['entity_id'])) {
+      $fileId = $file['id'];
+      $filepath = pathinfo($file['values'][$fileId]['path']);
+      $result = civicrm_api4('Contact', 'update', [
+        'values' => [
+          'id' => $attachmentParams['entity_id'],
+          'image_URL' => sprintf('/civicrm/contact/imagefile?photo=%s',$filepath['basename']),
```

Comment:
@JSProffitt I believe the style-checker was complaining because there's no space after the comma.